src/super_gradients/training/pipelines/pipelines.py

```diff
@@ -377,3 +380,63 @@ def _combine_image_prediction_to_video(
     ) -> VideoPoseEstimationPrediction:
         images_predictions = [image_predictions for image_predictions in tqdm(images_predictions, total=n_images, desc="Predicting Video")]
         return VideoPoseEstimationPrediction(_images_prediction_lst=images_predictions, fps=fps)
+
+
+class ClassificationPipeline(Pipeline):
+    """Pipeline specifically designed for Image Classification tasks.
+    The pipeline includes loading images, preprocessing, prediction, and postprocessing.
+
+    :param model:                       The classification model (instance of SgModule) used for making predictions.
+    :param class_names:                 List of class names corresponding to the model's output classes.
+    :param image_processor:             Single image processor or a list of image processors for preprocessing and postprocessing the images.
+    :param device:                      The device on which the model will be run. If None, will run on current model device. Use "cuda" for GPU support.
+    :param fuse_model:                  If True, create a copy of the model, and fuse some of its layers to increase performance. This increases memory usage.
+    """
+
+    def __init__(
+        self,
+        model: SgModule,
+        class_names: List[str],
+        device: Optional[str] = None,
+        image_processor: Optional[Processing] = None,
+        fuse_model: bool = True,
+    ):
+        super().__init__(model=model, device=device, image_processor=image_processor, class_names=class_names, fuse_model=fuse_model)
```

Comment:
Should the model be `BaseClassifier` instead? (Same for doc ` The classification model (instance of SgModule)`)

Reply:
`model: BaseClassifier` instead of `model: SgModule`, no?